packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/Relations.tsx

```diff
@@ -1064,7 +1064,7 @@ const ListItem = ({ data, index, style }: ListItemProps) => {
               </IconButton>
             ) : null}
             <Flex width="100%" minWidth={0} justifyContent="space-between">
-              <Box minWidth={0} paddingTop={1} paddingBottom={1} paddingRight={4}>
+              <Box minWidth={0} paddingTop={1} paddingBottom={1} marginRight={4} overflow="hidden">
```

Comment:
Is overflow necessary?  I think paddingRight still gives you the same result and I think it would be better since we are applying padding everywhere. 

The more I look at this though I'm thinking we shouldn't have this Box wrapper here. Instead shouldn't we handle the styling of the trigger (including the ellipsis) inside the RelationModalForm where we have the trigger?